src/js/api.ts

```diff
@@ -102,38 +119,48 @@ export async function loadPackagesFromImports(
  * Runs Python code using `PyCF_ALLOW_TOP_LEVEL_AWAIT
  * <https://docs.python.org/3/library/ast.html?highlight=pycf_allow_top_level_await#ast.PyCF_ALLOW_TOP_LEVEL_AWAIT>`_.
  *
- * .. admonition:: Python imports
- *    :class: warning
+ * .. admonition:: Python imports :class: warning
  *
  *    Since pyodide 0.18.0, you must call :js:func:`loadPackagesFromImports` to
- *    import any python packages referenced via `import` statements in your code.
- *    This function will no longer do it for you.
+ *    import any python packages referenced via `import` statements in your
+ *    code. This function will no longer do it for you.
  *
  * For example:
  *
  * .. code-block:: pyodide
  *
- *    let result = await pyodide.runPythonAsync(`
- *        from js import fetch
- *        response = await fetch("./packages.json")
- *        packages = await response.json()
- *        # If final statement is an expression, its value is returned to JavaScript
+ *    let result = await pyodide.runPythonAsync(` from js import fetch response
+ *        = await fetch("./packages.json") packages = await response.json() # If
+ *        final statement is an expression, its value is returned to JavaScript
```

Comment:
Is this automatic-fix by prettier? It looks like this breaks [formatting in docs](https://pyodide--2300.org.readthedocs.build/en/2300/usage/api/js-api.html#pyodide.runPythonAsync).

Reply:
No, that was my text editor's hardwrapper, which I probably applied manually without paying much attention. I think prettier leaves comments alone.